sunscreen_runtime/src/debugger/data.rs

```diff
@@ -305,11 +305,23 @@ fn create_plaintext_from_seal_data(pt: SealPlaintext, pk: &PrivateKey) -> InnerP
     sunscreen_plaintext.inner
 }
 
-#[derive(Clone, Serialize, Deserialize)]
+#[derive(Clone, Deserialize)]
 pub struct ZkpNodeType {
     // Send `BigInt` values as strings
     pub value: String,
```

Comment:
This should just be a `BigInt` type, and then you can serialize it as a string in the serialization impl.